scripts/java-api-check.sh

```diff
@@ -50,10 +44,9 @@ PREVIOUS_RELEASE=${TMPDIR}/previous/sdk-release.aar
 CURRENT_RELEASE_DIR=$(dirname "${CURRENT_RELEASE}")
 PREVIOUS_RELEASE_DIR=$(dirname "${PREVIOUS_RELEASE}")
 
-gh auth login --with-token < ./gh_token.txt
-LAST_VERSION_TAG=$(gh release list -L 1) #android-v10.3.0
-LAST_VERSION_TAG_ARRAY=($LAST_VERSION_TAG)
-LAST_VERSION=${LAST_VERSION_TAG_ARRAY[0]:9}
+LAST_VERSION_TAG=$(grep -o 'LAST_STABLE_VERSION_NAME=[0-9]*\.[0-9]*\.[0-9]*' ./gradle.properties) #10.5.0
```

Comment:
Thinking about using `gh list releases` instead of adding new property that needs to be managed manually? Can we just fix the way `gh` finds latest release? E.g. detect what branch is checked, then filter `gh list releases` differently?